app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentPresenter.kt

```diff
@@ -31,7 +31,8 @@ import javax.inject.Inject
 @FragmentScope
 class AdministratorControlsFragmentPresenter @Inject constructor(
   private val activity: AppCompatActivity,
-  private val fragment: Fragment
+  private val fragment: Fragment,
+  private val multiTypeBuilder: BindableAdapter.MultiTypeBuilder.Factory
```

Comment:
For consistency: `private val multiTypeBuilderFactory: `

Reply:
Updated.